crates/fluvio-cluster/src/runtime/local/spu.rs

```diff
@@ -2,6 +2,7 @@ use std::process::{Command, Stdio};
 use std::{fs::File, path::PathBuf};
 
 use anyhow::{Result as AnyResult, anyhow};
+use fluvio_types::defaults::SPU_PUBLIC_PORT;
 use tracing::{debug, info, instrument};
 
 use fluvio_controlplane_metadata::spu::{Endpoint, IngressAddr, IngressPort, SpuSpec, SpuType};
```

Comment:
```suggestion
use tracing::{debug, info, instrument};

use fluvio_types::defaults::SPU_PUBLIC_PORT;
use fluvio_controlplane_metadata::spu::{Endpoint, IngressAddr, IngressPort, SpuSpec, SpuType};
```

Reply:
This is nitpick, but we try to group together local workspace imports!

Reply:
Yes, Fluvio standard is 
- std lib
- third party
- fluvio crates